### PR TITLE
feat(DeviceManager): Get url for latest released firmware for huddly device

### PR DESCRIPTION
**Pull request by brikendr**
Use the SDK to find out what is the latest released firmware for any huddly device (GO and IQ for
now). Based on what device is plugged into your machine, calling `getLatestFirmwareUrl` will return
the url to the firmware package for the provided release channel (which by default is the "stable"
channel)

### CODE REVIEW

File: src/interfaces/ReleaseChannelEnum.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+enum ReleaseChannel {
+  STABLE = 'stable',
```
Comment (jontore) [inline src/interfaces/ReleaseChannelEnum.ts]: There is no need to include alpha/beta here.

File: tests/components/api.spec.ts [hunk 2]
```diff
@@ -493,4 +494,109 @@ describe('API', () => {
       expect(status).to.equals('enabled');
     });
   });
+
+  describe('#getLatestFirmwareUrl', () => {
```
Comment (jontore) [inline tests/components/api.spec.ts]: Nice

**Comment by jontore** [comment]
Coud be a nice iprovement if you can pass the url to the upgrader method.

**Comment by brikendr** [comment]
@jontore I did not write any "How To" guides in this feature branch since there is already a guide on how to perform firmware upgrade using the upgrader interface.

**Pull request merged**